spec/classes/prometheus_spec.rb

```diff
@@ -106,7 +106,7 @@
             )
           }
 
-          if ['centos-7-x86_64', 'centos-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'ubuntu-16.04-x86_64', 'ubuntu-18.04-x86_64', 'virtuozzolinux-7-x86_64'].include?(os)
+          if ['centos-7-x86_64', 'centos-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'ubuntu-16.04-x86_64', 'ubuntu-18.04-x86_64', 'ubuntu-20.04-x86_64', 'virtuozzolinux-7-x86_64'].include?(os)
```

Comment:
There must be a better way to indicate which ones use systemd?

Reply:
cleaned them all up